modules/gapi/test/common/gapi_imgproc_tests.hpp

```diff
@@ -52,6 +52,8 @@ GAPI_TEST_FIXTURE(SobelXYTest, initMatrixRandN, FIXTURE_API(CompareMats,int,int,
 GAPI_TEST_FIXTURE(EqHistTest, initMatrixRandN, FIXTURE_API(CompareMats), 1, cmpF)
 GAPI_TEST_FIXTURE(CannyTest, initMatrixRandN, FIXTURE_API(CompareMats,double,double,int,bool), 5,
     cmpF, thrLow, thrUp, apSize, l2gr)
+GAPI_TEST_FIXTURE(GoodFeaturesTest, initMatrixRandN, FIXTURE_API(CompareMats,int,double,double,int,bool,double), 7,
```

Comment:
Please use another input image to test a corner detector

Reply:
done